File: rei-do-chopp-site/src/app/orders/models/requests/order-product-list.request.ts
import { PaginationRequest } from "../../../shared/models/requests/pagination.request";

export class OrderProductListRequest extends PaginationRequest {
  OrderId: number

  constructor(params: Partial<OrderProductListRequest> ) {
    super(params);
    this.OrderId = params.OrderId || null;
  }
}

File: rei-do-chopp-api/ReiDoChopp.DataTransfer/OrdersProducts/Requests/OrderProductListRequest.cs
using System;
using ReiDoChopp.Domain.Utils.Enums;
using ReiDoChopp.DataTransfer.Pagination.Responses;
using ReiDoChopp.DataTransfer.Pagination.Requests;

namespace ReiDoChopp.DataTransfer.OrdersProducts.Request
{
    public class OrderProductListRequest : PaginationRequest
    {
        public int? Id { get; set; }
        public int? OrderId { get; set; }
        public int? ProductId { get; set; }
        public int? Quantity { get; set; }
        public double? UnitPriceCharged { get; set; }


        public OrderProductListRequest() : base(pageSize: 10, ordenationField: "Id", ordenationType: OrdenationTypeEnum.ASC) {}
    }
}